lib/ansible/module_utils/common/text/converters.py

```diff
@@ -209,6 +209,10 @@ def to_text(obj, encoding='utf-8', errors=None, nonstring='simplerepr'):
         else:
             errors = 'replace'
 
+    if isinstance(obj, bool):
+        # Return JSON/YAML 1.2 standardized true/false values
+        return "true" if obj else "false"
+
```

Comment:
This is definitely the wrong place for this to happen.  This function should not concern itself with YAML serialization.  This is basically just Python conversion to text type, not special case modifications to non python types.

Reply:
@sivel given your deep knowledge and experience with ansible, could you take a few minutes to get me headed in the right direction? A couple pointers into the code would be great.  I really want to get this right.

As part of mentioned change, we want to ensure that we only display booleans as `true`, `false` to the user, including output of `ansible-doc <module>`.

Reply:
You will likely have to make modifications in every location that outputs the value.

In the case of `ansible-doc`, the change would be required in `lib/ansible/cli/doc.py::DocCLI.add_fields`.  A new helper function might be required to avoid duplication.  The change likely needs to happen for the default value, as well as choices.

Something like:

```
default = "[Default: %s" % to_text(_coerce_bool_to_yaml(opt.pop('default', '(null)'))) + "]"
```